### DISCUSSION TITLE
pushImage wrong colors

### DISCUSSION THREAD

**Discussion by ameliaamelia** [Bodmer/TFT_eSPI#1890] [Alerts]
version: 2.4.61
ESP32
ILI9341

The command:

 **_tft.pushImage(101, 51, 96, 48, HeaterImg96);_**

produces  wrong colors, for example **_MAGENTA 0xF81F_**  is rendered as near **_CYAN 0x07FF_**
MAGENTA inverted  #0x1FF8.

TFT_eESPI.cpp the line 1464: buffer[j] = **_pgm_read_word(&data[i * w + j]);_**

must invert the two bytes;

**Comment by Bodmer** [comment]
Use setSwapBytes(true or false) to bontrol endianess management of different firmat images.
